### PR TITLE
P0 button

### PR DESCRIPTION
**Pull request by anshulraheja**
Button Component
1. Normal Button 
2. Outline Button
3. Floating Button

### CODE REVIEW

**Comment by m26unkwn** [comment]
You can give some padding to the Heading of a variant of component and the actual component style.

This is your Styling
image

This is what I wanted to say you can do.

image

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    --primary-btn-hover-color: #5998cd;
+    --success-btn-color: #65db65;
+    --success-btn-hover-color: #89da89;
+    --error-btn-color: #d43f3a;
+    --error-btn-hover-color:#d46360;
+    --link-btn-color: #3137e1;
+    --link-btn-hover-color: #6569df;
+    --float-btn-color:#0d5be1;
+    --float-btn-shadow-color:#9391df;
+    --float-btn-hover-color: #121124;
```
Comment (pankaj-prog) [inline components/button/button.css]: use capitalized variables

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    color: var(--error-btn-color);
+}
+.btn-outline-error:hover{
+    background: var(--error-btn-color);
+    color: var(--text-color);
+}
+.btn-outline-success{
+    border: 1px solid var(--success-btn-color);
+    color: var(--success-btn-color);
+}
```
Comment (pankaj-prog) [inline components/button/button.css]: increase the contrast. text is not readable

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+.btn-floating:hover{
+    background: var(--float-btn-hover-color);
+}
+.button-example-container{
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    flex-wrap: wrap;
+}
```
Comment (pankaj-prog) [inline components/button/button.css]: this CSS should be in the documentation stylesheet, as this style will be used in cdn link

**Review by pankaj-prog — COMMENTED** [review]
button looks good.

**Comment by PuneetP16** [comment]
image
On the button hover, the primary button becomes unreadable.

**Pull request merged**